grpc-proto/proto/query.proto

```diff
@@ -86,14 +74,13 @@ message Value {
     double double = 5; // double
     bool boolean = 6; // boolean
     string string = 7; // ascii, varchar, text
-    bytes bytes = 8; // blob, custom
-    Inet inet = 9; // inet
-    Uuid uuid = 10; // uuid, timeuuid
-    Date date = 11; // date
-    Time time = 12; // time
-    Collection collection = 13; // list, set, tuple
-    Map map = 14;
-    UdtValue udt = 15;
+    bytes bytes = 8; // blob, inet, custom
```

Comment:
Would it be meaningful to split blob, inet and custom into distinct cases even though each of them will end up being represented as `bytes`?

Reply:
It's best to have the minimum number of fields as possible: https://developers.google.com/protocol-buffers/docs/overview#assigning_field_numbers

Reply:
TIL :+1: